ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committing/ProcessTransactionSubmission.scala

```diff
@@ -315,7 +318,11 @@ private[kvutils] class ProcessTransactionSubmission(
     if (causalKeyMonotonicity)
       pass
     else
-      reject(recordTime, buildRejectionLogEntry(transactionEntry, RejectionReason.Inconsistent))
+      reject(
+        recordTime,
+        buildRejectionLogEntry(
+          transactionEntry,
+          RejectionReason.Inconsistent("Causal Monotonicity Violated")))
```

Comment:
```suggestion
          RejectionReason.Inconsistent("Causal monotonicity violated")))
```

Reply:
Please apply this change as well for consistency.